src/Microsoft.AspNet.Mvc.Core/MvcOptions.cs

```diff
@@ -163,5 +165,28 @@ public int MaxModelValidationErrors
         /// </ul>
         /// </remarks>
         public IList<IMetadataDetailsProvider> ModelMetadataDetailsProviders { get; }
+
+        /// <summary>
+        /// Provides programmatic configuration for the html helpers and view context.
+        /// </summary>
+        public HtmlHelperOptions HtmlHelperOptions
+        {
+            get
+            {
+                return _htmlHelperOptions;
+            }
+
+            set
+            {
+                if (value == null)
+                {
+                    throw new ArgumentNullException(
+                        nameof(value),
+                        Resources.FormatPropertyOfTypeCannotBeNull(nameof(HtmlHelperOptions), typeof(MvcOptions)));
+                }
```

Comment:
Also `FormatPropertyOfTypeCannotBeNull` is for when your code needs to access a property of a separate object.